src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -430,7 +429,9 @@ export const CheckoutPayment = ({
 
   const handleClickBackButton = () => {
     trackOfferEvent({ eventName: EventName.ContactInformationPageGoBack })
+    history.push(detailsPageLink)
   }
+  const detailsPageLink = `/${locale.path}/new-member/checkout/details/${quoteCartId}`
```

Comment:
Let's move this inside the `handleClickBackButton` function. Looks odd that it's defined after the function :/